### PR TITLE
Refactor for f strings.

### PR DESCRIPTION
**Pull request by changeling**
Left the ``deprecated.py` alone for now, but it perhaps that deserves re-factoring for a very minor performance win?

Skipping some of `relay/tests/test_connection_query.py` for nested quoting laziness on my part.

### CODE REVIEW

**Comment by ekampf** [comment]
Your `black` pre-commit is failing

**Comment by ekampf** [comment]
Also please rebase from `next/master`

File: graphene/types/base.py [hunk 1]
```diff
@@ -20,10 +20,10 @@ def __setattr__(self, name, value):
         if not self._frozen:
             super(BaseOptions, self).__setattr__(name, value)
         else:
-            raise Exception("Can't modify frozen Options {}".format(self))
+            raise Exception(f"Can't modify frozen Options {self}")
 
     def __repr__(self):
-        return "<{} name={}>".format(self.__class__.__name__, repr(self.name))
+        return f"<{self.__class__.__name__} name={repr(self.name)}>"
```
Comment (ekampf) [inline graphene/types/base.py]: why `repr(self.name)` and not just `self.name!r` ?

**Comment by dvndrsn** [comment]
F-Strings are not supported in Python 2.7, so this PR would need to be merged in 3.0 when 2.7 and 3.4 support is dropped. Making sure this is tagged appropriately.

**Comment by changeling** [comment]
> F-Strings are not supported in Python 2.7, so this PR would need to be merged in 3.0 when 2.7 and 3.4 support is dropped. Making sure this is tagged appropriately.

Good note, and thanks. I'd not thought to do that, as it's targeted at `next/master`, which is 3 only, but I'll do that going forward.

**Comment by changeling** [comment]
> Per our discussion on slack, lets use this opportunity to consistently use either
> `f"{repr(value)}` or f"{value!r}".
> We decided on `repr` in Slack...

@ekampf, "View changes" doesn't offer changes, so I'm unable to mark this change request as resolved.

File: docs/relay/nodes.rst [hunk 2]
```diff
@@ -52,7 +52,7 @@ Example of a custom node:
 
         @staticmethod
         def to_global_id(type, id):
-            return '{}:{}'.format(type, id)
+            return f'{type}:{id}'
```
Comment (jkimbo) [inline docs/relay/nodes.rst]: Can we get black to run on the examples in the docs as well? In this case the quotes should be double quotes.

File: graphene/types/tests/test_structures.py [hunk 3]
```diff
@@ -19,7 +19,7 @@ def test_list_with_unmounted_type():
 
     assert (
         str(exc_info.value)
-        == "List could not have a mounted String() as inner type. Try with List(String)."
+        == "List can not have a mounted String() as inner type. Try List(String)."
```
Comment (jkimbo) [inline graphene/types/tests/test_structures.py]:
```suggestion
        == "List cannot have a mounted String() as inner type. Try List(String)."
```

File: graphene/types/tests/test_structures.py [hunk 4]
```diff
@@ -97,7 +97,7 @@ def test_nonnull_with_unmounted_type():
 
     assert (
         str(exc_info.value)
-        == "NonNull could not have a mounted String() as inner type. Try with NonNull(String)."
+        == "NonNull can not have a mounted String() as inner type. Try NonNull(String)."
```
Comment (jkimbo) [inline graphene/types/tests/test_structures.py]:
```suggestion
        == "NonNull cannot have a mounted String() as inner type. Try NonNull(String)."
```

File: graphene/types/tests/test_structures.py (hunk 3 above)
Comment (changeling) [reply graphene/types/tests/test_structures.py]: I'll defer to the more common usage here. :) `cannot` it is.

Also changing in docstrings in `graphene/pyutils/signature.py`, lines 811 and 818.

File: docs/relay/nodes.rst (hunk 2 above)
Comment (changeling) [reply docs/relay/nodes.rst]: I've been trying to blacken them all, but clearly missed some. Lemme know if there are others. I'll do another run through.

Comment (changeling) [reply docs/relay/nodes.rst]: NM. This is addressed in https://github.com/graphql-python/graphene/pull/992

**Comment by changeling** [comment]
This is gonna need to wait until we decide on 3.5 support.

**Comment by changeling** [comment]
Lookd like we have concensus on 3.6+, so this is ready to go (or review more)!

**Comment by changeling** [comment]
What's the status here? Requested changes made, though the system still shows `Changes requested`. I believe that's your https://github.com/graphql-python/graphene/pull/984#pullrequestreview-244607933, @ekampf.

Can this be merged/reviewed so I can clear this from my end?

**Comment by changeling** [comment]
Thoughts on reviewing/merging this, @ProjectCheshire, @ekampf, @dan98765 ?

**Comment by KingDarBoja** [comment]
Any updates on this PR? Just curious as the author has no activity since July 2019 😨

**Comment by ekampf** [comment]
@KingDarBoja its kind of stuck on conflicts it seem...  @changeling  ?

**Comment by syrusakbary** [comment]
Apologizes for the delay actioning this PR.

This PR is taking the next steps to replace f-strings: https://github.com/graphql-python/graphene/pull/1158

**Pull request closed without merging**